fusionfusion/annotationFunction.py

```diff
@@ -190,8 +190,12 @@ def filterAndAnnotation(inputFilePath, outputFilePath, genome_id, is_grc):
 
         if filter_same_gene == True and sameGeneFlag == 1: continue
 
-        print('\t'.join(F[0:8]) + '\t' + ';'.join(gene1) + '\t' + ';'.join(junction1) + '\t' + ';'.join(gene2) + '\t' + ';'.join(junction2) + '\t' + \
-              F[11] + '\t' + F[12] + '\t' + F[16] + '\t' + F[17], file = hOUT)
+        line = '\t'.join(F[0:8]) + '\t' \
+            + ';'.join(gene1) + '\t' + ';'.join(junction1) + '\t' \
+            + ';'.join(gene2) + '\t' + ';'.join(junction2)
+        if includes_scores:
+            line += '\t' + F[11] + '\t' + F[12] + '\t' + F[16] + '\t' + F[17]
+        print(line, file = hOUT)
```

Comment:
https://www.python.org/dev/peps/pep-0008/#other-recommendations

```suggestion
        print(line, file=hOUT)
```